tests/chainer_tests/functions_tests/noise_tests/test_simplified_dropconnect.py

```diff
@@ -98,4 +103,88 @@ def test_backward_gpu_nobias(self):
                             None, cuda.to_gpu(self.gy))
 
 
+class TestSimplifiedDropconnectBatchwiseMaskShape(unittest.TestCase):
+
+    def setUp(self):
+        self.W = numpy.random.uniform(
+            -1, 1, (2, 3)).astype(numpy.float32)
+        self.b = numpy.random.uniform(
+            -1, 1, 2).astype(numpy.float32)
+
+        self.x = numpy.random.uniform(-1, 1, (4, 3)).astype(numpy.float32)
+        self.gy = numpy.random.uniform(-1, 1, (4, 2)).astype(numpy.float32)
+        self.y = self.x.dot(self.W.T) + self.b
+        self.check_forward_options = {}
+        self.check_backward_options = {}
+
+    def check_forward(self, x_data, W_data, b_data):
+        # Check only data type, y is tested by SimplifiedDropconnect link test.
+        x = chainer.Variable(x_data)
+        W = chainer.Variable(W_data)
+        b = chainer.Variable(b_data)
+        func = simplified_dropconnect.SimplifiedDropconnect(0.5, None, True)
+        y = func(x, W, b)
+        self.assertEqual(y.data.dtype, numpy.float32)
+        self.assertEqual(func.mask.shape, (x.shape[0],) + W.shape)
+
+    def test_forward_cpu(self):
+        self.check_forward(self.x, self.W, self.b)
+
+    @attr.gpu
+    def test_forward_gpu(self):
+        self.check_forward(
+            cuda.to_gpu(self.x), cuda.to_gpu(self.W), cuda.to_gpu(self.b))
+
+
+class TestSimplifiedDropconnectNotBatchwiseMask(unittest.TestCase):
```

Comment:
I would like to confirm my understanding. The difference of `TestSimplifiedDropconnect` and `TestSimplifiedDropconnectBatchwiseMaskShape`  is to use the default argument of `batchwise_mask` or not, isn't it?